core/sql/executor/ExStats.cpp

```diff
@@ -3678,9 +3292,8 @@ void ExPartitionAccessStats::getVariableStatsInfo(char * dataBuffer,
 	      "AnsiName: %s PhysName: %s BuffersSize: %u BuffersSent: %u BuffersRcvd: %u NumMessages: %Ld MsgBytes: %Ld StatsBytes: %Ld MsgBytesSent: %Ld MsgBytesRcvd: %Ld ",
 	      ansiName_, fileName_,
 	      bufferStats()->sendBufferSize(), bufferStats()->sentBuffers().entryCnt(), bufferStats()->recdBuffers().entryCnt(), 
-	      fsDp2MsgsStats()->getNumMessages(),
-	      fsDp2MsgsStats()->getMessageBytes(),
-	      fsDp2MsgsStats()->getStatsBytes(),
+	      exeSEStats()->getNumIOCalls(),
+	      exeSEStats()->getNumIOBytes(),
```

Comment:
Yes.  It is supposed to denote how many stats bytes are transferred from SE to the executor periodically. But with the new SE, this concept is no longer applicable.